lib/cryptoexchange/exchanges/cex/services/market.rb

```diff
@@ -0,0 +1,40 @@
+module Cryptoexchange::Exchanges
+  module Cex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Cex::Market::API_URL}/ticker/#{market_pair.target}/#{market_pair.base}"
```

Comment:
After the base/target is fixed https://github.com/coingecko/cryptoexchange/pull/214/files/c909843ceccb16d73c1eba1080342e78cf58ff21..092e895da9438098c6f743b40d2c3f7ffdc810bc#r152446208, passing of the base/target should be fixed for this ticker url as well

Reply:
base/target position corrected